docusaurus.config.ts

```diff
@@ -187,8 +187,12 @@ const config: Config = {
         id: 'theoads',
         path: 'theoads',
         routeBasePath: '/theoads',
-        sidebarPath: './sidebarsTHEOads.ts',
+        sidebarPath: './sidebarsOptiViewAds.ts',
```

Comment:
Nit-pick: maybe just `sidebarsAds.ts`? Otherwise every file will be called "OptiView". 😅